config/manager/metadata-server.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: o2ims.oran.openshift.io/v1alpha1
+kind: Inventory
+metadata:
+  annotations:
+  labels:
+    app.kubernetes.io/created-by: oran-o2ims
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: orano2ims
+    app.kubernetes.io/part-of: oran-o2ims
+  name: metadata-server
+  namespace: oran-o2ims
+spec:
+  alarmSubscriptionServerConfig:
+    enabled: false
+  cloudId: ae6bd821-a4a4-4e53-89d1-94609968e40e
```

Comment:
This `cloudId` is meant to be the `globalCloudId`... as discussed internally this should be made optionally and excluded from this CR and only provided by the end user when the system is ready to be registered with the SMO, this is because the system cannot register with the SMO until the administrator has setup this cloud in the SMO and has acquired the globalCloudId.  

So i think what should happen here is that this field be made optional, and renamed to `globalCloudId` so that its intent is clear.